edx_proctoring/templates/proctoring/seq_proctored_exam_entrance.html

```diff
@@ -0,0 +1,75 @@
+{% load i18n %}
+<div class="sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Would you like to take {{ display_name }} as a Proctored Exam?
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since you're enrolled in this course as a verified student, you have the option to take this exam with
+      online proctoring. Online proctoring is one requirement towards being eligible for credit.
+    {% endblocktrans %}
+  </p>
+  <div class="gated-sequence">
+    <span><i class="fa fa-lock"></i></span>
+    <a class="start-timed-exam" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=true data-start-immediately=false>
+      {% trans "Yes, take this as a proctored exam (and be eligible for credit)" %}
+    </a>
+    <p>
+      {% blocktrans %}
+        You will need to <strong>download and install {{platform_name}} approved software </strong>for the online proctoring of
```

Comment:
"You will be guided through installing edX-approved online proctoring software and performing various checks to set up your proctored exam session. Have your photo ID ready for the photo ID verification step.
Immediately after you complete the set up, you will begin your timed and proctored exam. 
Before you set up your proctoring session, you might want to read {link to student-facing doc HTML topic} to learn more about taking a proctored exam."